kafka-ui-contract/src/main/resources/swagger/kafka-connect-api.yaml

```diff
@@ -53,6 +53,28 @@ paths:
         500:
           description: Internal server error
 
+  /connectors/filtered:
```

Comment:
Could we add this filter to /connectors endpoint and check it for null value? I don't see any reason to have separate endpoint for filtering. We can provide backward compatibility with current endpoint

Reply:
Will do.

Reply:
Done.